test/integration/forms/action.js

```diff
@@ -1988,4 +1988,375 @@ context('Patient Action Form', function() {
+            form: { data: { id: '22222' } },
+            files: { data: null },
+          },
+        };
+
+        return fx;
+      })
+      .routePatientFlows(fx => {
+        fx.data = [];
+
+        return fx;
+      })
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.owner.data = { id: '11111', type: 'clinicians' };
+        fx.data.relationships.form.data = { id: '11111' };
+
+        fx.data.relationships['form-responses'].data = [];
+
+        return fx;
+      })
+      .routeFormActionFields(fx => {
+        fx.data.attributes = { patient: { fields: { foo: 'bar' } } };
+
+        return fx;
+      })
+      .routePatientByAction()
+      .routeFormByAction(_.identity, '11111')
+      .routeLatestFormResponse()
+      .routeFormDefinition()
+      .visit('/patient/dashboard/1')
+      .wait('@routePatient')
+      .wait('@routePatientActions')
+      .wait('@routePatientFlows');
+
+    cy
+      .get('.patient__list')
+      .find('.table-list__item')
+      .as('listItems')
+      .first()
+      .find('[data-form-region] button')
+      .click()
+      .wait('@routeAction')
+      .wait('@routeFormByAction')
+      .wait('@routePatientByAction');
+
+    cy
+      .get('.form__controls')
+      .find('.js-save-button')
+      .should('exist');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[familyHistory]"]')
+      .should('exist');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[storyTime]"]')
+      .should('exist');
+
+    cy
+      .iframe()
+      .find('[name="data[patient.fields.foo]"]')
+      .should('have.value', 'bar')
+      .should('not.be.disabled');
+
+    cy
+      .get('.form__context-trail')
+      .find('.js-back')
+      .click()
+      .wait('@routePatient')
+      .wait('@routePatientActions')
+      .wait('@routePatientFlows');
+
+    cy
+      .routeAction(fx => {
+        fx.data.id = '2';
+        fx.data.relationships.owner.data = { id: '22222', type: 'clinicians' };
+        fx.data.relationships.form.data = { id: '22222' };
+
+        return fx;
+      });
+
+    cy
+      .get('.patient__list')
+      .find('.table-list__item')
+      .as('listItems')
+      .last()
+      .find('[data-form-region] button')
+      .click();
+
+    cy
+      .wait('@routeAction')
+      .wait('@routeFormByAction')
+      .wait('@routePatientByAction');
+
+    cy
+      .get('.form__controls')
+      .find('.form__locked')
+      .should('contain', 'You don’t have permission to edit or submit this form.');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[familyHistory]"]')
+      .should('not.exist');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[storyTime]"]')
+      .should('not.exist');
+
+    cy
+      .iframe()
+      .find('[name="data[patient.fields.foo]"]')
+      .should('have.value', 'bar')
+      .should('be.disabled');
+  });
+
+  specify('user has only work:team:submit permission', function() {
```

Comment:
Quick test overview:

1. Set the current clinician to have the `work:team:submit` permission.
2. Load a patient dashboard with two actions. One that is owned by the current clinician's team and another that is not.
3. Then I click through to both of those action's forms and verify that the submit button is properly restricted. And that the form was rendered as read only.

Reply:
In this test, I need to also verify the submit button is enabled for an action owned by a fellow team member. That's the reason for the coverage drop in the `canSubmit()` function.